project/endpoints/books.py

```
@@ -0,0 +1,18 @@
+from fastapi import FastAPI
+from fastapi import HTTPException
+from project.services.book_loader import book_loader
+from project.models.book import Book
+
+app = FastAPI()
+books = book_loader()
+
+@app.get("/books")
+def get_books():
+    return books
+
+@app.get("/books/{isbn}")
+def find_book_by_isbn(isbn: str) -> Book:
+    try:
+        return books[isbn]
+    except:
```

⏳ Sand / Non-blocking but requires future consideration
```except:``` will catch all exceptions including things like ```KeyboardInterrupt``` or ```SystemExit``` possibly resulting in miscomprehension of the actual error catched since the error message will always be a 404 Not Found. To handle this more gracefully I suggest you use ```except KeyError``` since that will be the error raised when ```return books[isbn]``` fails.